tests/openedx_tagging/core/tagging/test_views.py

```diff
@@ -174,8 +184,13 @@ def test_list_taxonomy(self, user_attr: str | None, expected_status: int, tags_c
                     "system_defined": False,
                     "visible_to_authors": True,
                     "tags_count": tags_count,
+                    # Enabled taxonomy can be modified by taxonomy admins
+                    "can_change_taxonomy": is_admin,
+                    "can_delete_taxonomy": is_admin,
+                    "can_tag_object": False,
```

Comment:
You'd think so, wouldn't you? But the base [`oel_tagging.can_change_object_tag_objectid`](https://github.com/openedx/openedx-learning/blob/jill/serialize-permissions/openedx_tagging/core/tagging/rules.py#L130-L137) always returns `False`, and so nobody has permission to add base ObjectTags.

I'm more than happy to change this, though, I think it's over cautious.

Reply:
So it's False because the admin could add a ContentObjectTag but not an ObjectTag? Is that what you mean?

If so, is there a similar API that includes `can_tag_content_object` or something? This field just doesn't seem useful at the moment.

Reply:
More context here: https://github.com/openedx/openedx-learning/pull/74#discussion_r1313168598

Reply:
@bradenmacdonald 

> So it's False because the admin could add a ContentObjectTag but not an ObjectTag? Is that what you mean?

Yes.

> If so, is there a similar API that includes `can_tag_content_object` or something? This field just doesn't seem useful at the moment.

https://github.com/openedx/edx-platform/pull/34004 shows where `can_tag_object` is True in the content_tagging context, does that help?

Reply:
Yes, that helps :) 
Could you please just add a comment here along those lines? Something like "This is False because we default to not allowing users to tag arbitrary objects. But specific uses of this code like content_tagging in edx-platform will override this perm to be true for their use cases"